modules/src/ics02_client/handler.rs

```diff
@@ -6,32 +6,32 @@ use crate::ics02_client::msgs::ClientMsg;
 use crate::ics24_host::identifier::ClientId;
 
 use crate::ics02_client::context::ClientReader;
-use crate::ics02_client::handler::create_client::CreateClientOutput;
-use crate::ics02_client::handler::update_client::UpdateClientOutput;
+use crate::ics02_client::handler::create_client as ics2_create_client;
+use crate::ics02_client::handler::update_client as ics2_update_client;
```

Comment:
```suggestion
```
can't you just use `create_client` and `update_client` brought in below?

Reply:
Hmm no, we cannot use the default name because it conflicts with the declarations:

```
pub mod create_client;
pub mod update_client;
```

The specific error would be:
```
A second item with name 'update_client' imported. Try to use an alias. [E0252]
```